specification/resources/resource-manager/Microsoft.Solutions/stable/2019-07-01/managedapplications.json

```diff
@@ -0,0 +1,2134 @@
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Jit request tags"
+        }
+      },
+      "description": "Information about JIT request."
+    },
+    "JitRequestDefinitionListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/JitRequestDefinition"
+          },
+          "description": "The array of Jit request definition."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to use for getting the next set of results."
+        }
+      },
+      "description": "List of JIT requests."
+    },
+    "JitRequestProperties": {
+      "properties": {
+        "ApplicationResourceId": {
+          "type": "string",
+          "description": "The parent application id."
+        },
+        "PublisherTenantId": {
+          "type": "string",
+          "description": "The publisher tenant id."
+        },
+        "JitAuthorizationPolicies": {
+          "description": "The JIT authorization policies.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/JitAuthorizationPolicies"
+          }
+        },
+        "JitSchedulingPolicy": {          
+          "$ref": "#/definitions/JitRequestProperties",
+          "description": "The JIT request properties."
+        },
+        "ProvisioningState": {
+          "readOnly": true,
+          "$ref": "#/definitions/ProvisioningState",
+          "description": "The JIT request provisioning state."
+        },
+        "JitRequestState": {
+          "readOnly": true,
+          "$ref": "#/definitions/JitRequestState",
+          "description": "The JIT request state."
+        },
+        "CreatedBy": {
+          "$ref": "#/definitions/ApplicationClientDetails",
+          "description": "The client entity that created the JIT request."
+        },
+        "UpdatedBy": {
+          "$ref": "#/definitions/ApplicationClientDetails",
+          "description": "The client entity that last updated the JIT request."
+        }
+      },
+      "required":  [
+        "ApplicationResourceId",
+        "JitAuthorizationPolicies",
+        "JitSchedulingPolicy"
+      ],
+      "description": "Information about JIT request properties"
+    },
+    "JitAuthorizationPolicies": {
+      "properties": {
+        "PrincipalId": {
+          "type": "string",
+          "description": "The the principal id that will be granted JIT access."
+        },
+        "RoleDefinitionId": {
+          "type": "string",
+          "description": "The role definition id that will be granted to the Principal."
+        }
+      },
+      "required": [
+        "PrincipalId",
+        "RoleDefinitionId"
+      ],
+      "description": "The JIT authorization policies."
+    },
+    "JitSchedulingPolicy": {
+      "properties": {
+        "Type": {
```

Comment:
can you call this "schedulingType" or similar so it is not confused with the top-level "type" property?

Reply:
This type should always be referenced as JitSchedulingPolicy.Type, it should be relatively safe.